save-core/src/commonMain/kotlin/org/cqfn/save/core/config/SaveProperties.kt

```diff
@@ -14,7 +14,7 @@ import kotlinx.serialization.Serializable
 /**
  * Configuration properties of save application, retrieved either from properties file
  * or from CLI args.
- * @property testConfig Path to a configuration of a test suite
+ * @property testConfigName Path to a configuration of a test suite
```

Comment:
Maybe `testConfigPath`, implying that it shouldn't simply be a file name?

Reply:
fixed